.github/workflows/test.yml

```
@@ -0,0 +1,58 @@
+name: Tests
+
+on:
+  # Uncomment this if build on every push is needed
+  # push:
+  #   branches-ignore: [main, beta, alpha] # skip for deployment branches.
+  #   tags-ignore: ['*']
+  pull_request:
+    branches: ['*']
```

```suggestion
on: [push, pull_request]
```

This part can be simplified. Also, I suggest tests on each push instead of just PRs. It helps catch issues sooner from my experience.

---

I think the name of the file is confusing, it should be `Test Builds`, and we would only want to build test builds for pull request opening/updating, etc, not when there is any commit in the repo. right?